bindings/python/aditofpython.cpp

```diff
@@ -53,7 +53,7 @@ PYBIND11_MODULE(aditofpython, m) {
         .def_readwrite("mode", &aditof::CameraDetails::mode)
         .def_readwrite("frameType", &aditof::CameraDetails::frameType)
         .def_readwrite("connection", &aditof::CameraDetails::connection)
-        .def_readwrite("range", &aditof::CameraDetails::range)
+        .def_readwrite("range", &aditof::CameraDetails::rangeMax)
```

Comment:
This line should be: .def_readwrite("maxDepth", &aditof::CameraDetails::maxDepth). There's no rangeMax anymore. The python bindings shouldn't even compile anymore.
Also, can you create another binding for minDepth as well?

Reply:
totally forgot about bindings. working on it now.

Reply:
changed. thank you.